threedi_modelchecker/config.py

```diff
@@ -2859,6 +2852,15 @@ def is_none_or_empty(col):
     )
     for col in vegetation_parameter_columns
 ]
+CHECKS += [
+    CrossSectionFloatListCheck(
+        error_code=87,
+        column=col,
+        shapes=(constants.CrossSectionShape.TABULATED_YZ,),
+    )
+    for col in vegetation_parameter_columns
```

Comment:
```suggestion
    for col in [*vegetation_parameter_columns, models.CrossSectionDefinition.friction_values]
```
if they use the same error code it would be nice if the checks are together